File: docs/components_page/components/popover/popover_components.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

popovers <- htmlDiv(
  list(
    dbcButton(
      "Click Me",
      id = "component-target", n_clicks = 0
    ),
    dbcPopover(
      list(
        dbcPopoverHeader("Popover header"),
        dbcPopoverBody("And here's some amazing content. Cool!")
      ),
      target = "component-target",
      trigger = "click"
    )
  )
)

File: docs/components_page/components/popover/popover_components.py
import dash_bootstrap_components as dbc
from dash import html

popovers = html.Div(
    [
        dbc.Button(
            "Click Me",
            id="component-target",
            n_clicks=0,
        ),
        dbc.Popover(
            [
                dbc.PopoverHeader("Popover header"),
                dbc.PopoverBody("And here's some amazing content. Cool!"),
            ],
            target="component-target",
            trigger="click",
        ),
    ]
)
